demo-client/src/main/java/com/illia/client/service/DemoService.java

```diff
@@ -0,0 +1,76 @@
+package com.illia.client.service;
+
+import com.illia.client.config.ClientConfig;
+import com.illia.client.http_client.MyHttpClient;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.*;
+
+
+@Slf4j
+@Service
+public class DemoService {
+
+    @Autowired
+    ClientConfig clientConfig;
+
+    @Autowired
+    MyHttpClient client;
```

Comment:
nit: remember to keep the private fields private, linter would pick up on this with ease